advisories/github-reviewed/2022/04/GHSA-cvx5-m8vg-vxgc/GHSA-cvx5-m8vg-vxgc.json

```diff
@@ -63,6 +63,10 @@
       "type": "WEB",
       "url": "https://jira.xwiki.org/browse/XWIKI-5168"
     },
+    {
+      "type": "WEB",
+      "url": "https://github.com/xwiki/xwiki-commons/commit/215951cfb0f808d0bf5b1097c9e7d1e503449ab8"
```

Comment:
Relevant change - patch commit for fix

Reply:
Any specific reason why you want to have the reference of the commit published? Usually we don't put the commit in the references in our advisories, generally because there might have several commits involved. Here it seems that it's indeed only one commit. Note that I'm ok with the change, just wondering why you want it.

Reply:
Yes, if it's some standard we missed, we could decide to enforce it in our process to create security advisories.